### PR TITLE
V2 snapshotting fixes

### PR DESCRIPTION
**Pull request by aendrew**
~This is still in-progress; I think the `<head>` section is missing a lot of meta tags, and I'm not sure the CTM check is working.~

~TODO, render `<HtmlHead>` to string and supply to template via HtmlWebpackPlugin.~

~Requires g-components@7.0.0-rc2.~

~Requires g-components@7.0.0-rc6. Ready to go otherwise. 👍~

Now uses g-components@^7.0.1

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -67,79 +72,83 @@ const App = context => {
...
-        mekæo iomete træ. Int co onjo finnlæƿðo subjunkċiō, kaj faka eblecō mīnimumē ōƿ. Ore verba
-        ðuonhoro komplēksā il, hierāŭæ propōzicio ÞīudīrēkÞen iz sur.
-      </p>
-
-      <p>
-        Mæl denta sūȝstǣnÞivo bv, ēhe stif armo duūmæ ōp. Ec Þet pluso traigi. Ē vēla lǣstæ
-        fiƿǽĵvorto kūn. Jesī kiomas duondifinǽ hej he, agæ færās malloƿġīgō go. Tripunkto
-        reciprōkeċo op ǽġā, eliġi eŭro postmorgǣŭ ul anc.
-      </p>
+      <main key="main" role="main">
```
Comment (tlfrd) [inline app/app.js:90]: v7-rc3 of g-components has a slightly different article layout component now. Maybe we should use v7-rc3 here too so we can simplify the layout here? https://github.com/Financial-Times/g-components/blob/v7/src/article-layout/index.js#L43

Comment (tlfrd) [reply app/app.js:90]: Or rc4

File: app/app.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ import React, { useEffect, useState } from 'react';
 import { hot } from 'react-hot-loader/root';
 import Layout from '@financial-times/g-components/layout';
 import ArticleHead from '@financial-times/g-components/article-head';
-import ArticleFooter from '@financial-times/g-components/article-footer';
+import Epilogue from '@financial-times/g-components/Epilogue';
```
Comment (tlfrd) [inline app/app.js]: Does it make a difference if the `Epilogue` in the path is uppercase?

**Comment by aendrew** [comment]
It's failing because Footer has relative links. Will need 7.0.2

**Pull request merged**